### PR TITLE
added gobench script

### PR DESCRIPTION
**Pull request by hkernbach**
needs some final adjustments when gobench script is finalized.

### CODE REVIEW

**Comment by hkernbach** [comment]
maybe find also a more elegant waiting function for the arangod process (currently 15s wait)

File: jenkins/runGobenchPerformance.fish [hunk 1]
```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env fish
+set -xg simple (pwd)/performance
+set -xg gobenchdir (pwd)/gobench
+set -xg date (date +%Y%m%d)
+set -xg datetime (date +%Y%m%d%H%M)
+
+if test -z "$ARANGODB_TEST_CONFIG"
+  set -xg ARANGODB_TEST_CONFIG run-small-edges.js
```
Comment (mchacki) [inline jenkins/runGobenchPerformance.fish]: I think this is not correct here.
That config is only relevant for simplePerfTest

File: jenkins/runGobenchPerformance.fish [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+      --javascript.app-path /tmp/app \
+      --javascript.startup-directory /ArangoDB/js \
+      --server.rest-server true \
+      --server.endpoint tcp://0.0.0.0:8529 \
+      --javascript.module-directory `pwd` \
+      --server.authentication false \
+      --log.foreground-tty \
+      /data/database & \
+      echo 'Waiting 15s for ArangoDB' && \
+      sleep 15 &&
```
Comment (mchacki) [inline jenkins/runGobenchPerformance.fish]: typically we loop here with curl requests to _api/version.
As soon as this is available the server runs.

Comment (mchacki) [reply jenkins/runGobenchPerformance.fish]: from @goedderz 

```

wait_for_arango() {
    local ENDPOINT="$1"

    set +x
    echo "...waiting for curl -s $ENDPOINT/_api/version"

    while ! curl -s "$ENDPOINT"/_api/version
    do
        sleep 0.1
    done

    set -x
}
```
you can hardcode $ENDPOINT here

Comment (goedderz) [reply jenkins/runGobenchPerformance.fish]: note that this is bash code, I don't know what's necessary to translate it to fish...

**Comment by hkernbach** [comment]
applied requested changes.

**Pull request merged**